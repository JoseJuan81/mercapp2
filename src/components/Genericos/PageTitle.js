import React from 'react';

export const PageTitle = React.memo(({ title }) => {

    return (
        <h1
            className="
                text-4xl font-bold text-warmGray-500
                text-center
                mb-4
            "
        >{ title }</h1>
    )
})
