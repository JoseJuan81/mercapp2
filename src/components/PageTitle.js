import React from 'react';

export const PageTitle = React.memo(({ title }) => {
    return (
        <h1
            className="
                text-2xl font-light text-warmGray-800
                text-center
                mb-4
            "
        >{ title }</h1>
    )
})
