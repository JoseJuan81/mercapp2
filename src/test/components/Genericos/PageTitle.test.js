import { shallow } from 'enzyme';

import { PageTitle } from "../../../components/Genericos/PageTitle"

describe('pruebas sobre archivo PageTitle.js', () => {

    test('se debe renderizar correctamente', () => {
        const wrapper = shallow(<PageTitle title="Hola mundo" />);

        expect( wrapper ).toMatchSnapshot();
        expect( wrapper.find('h1').text()).toBe('Hola mundo');
    })
})