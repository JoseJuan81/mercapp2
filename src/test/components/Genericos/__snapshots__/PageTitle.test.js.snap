// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas sobre archivo PageTitle.js se debe renderizar correctamente 1`] = `
<h1
  className=" text-4xl font-bold text-warmGray-500 text-center mb-4 "
>
  Hola mundo
</h1>
`;
